kount/util/xml_rules.py

```diff
@@ -3,17 +3,21 @@
 # This file is part of the Kount python sdk project
 # https://github.com/Kount/kount-ris-python-sdk/)
 # Copyright (C) 2017 Kount Inc. All Rights Reserved.
-"""generated from xmlparser.py, must be updated if
-changed xml_filename = 'validate.xml' from settings.py"""
-from __future__ import (
-    absolute_import, unicode_literals, division, print_function)
+"""autogenerated rules from xml_filename = 'validate.xml' in settings.py"""
+from __future__ import absolute_import, unicode_literals, division, print_function
+import os
+from kount.util.xmlparser import xml_to_dict
+from kount.settings import RESOURCE_FOLDER, XML_FILENAME
+
 
 __author__ = "Yordanka Spahieva"
 __version__ = "1.0.0"
 __maintainer__ = "Yordanka Spahieva"
 __email__ = "yordanka.spahieva@sirma.bg"
 __status__ = "Development"
 
+XML_FILENAME_PATH = os.path.join(os.path.dirname(__file__), '..', '..',
+                                 RESOURCE_FOLDER, XML_FILENAME)
 
 XML_DICT = {
```

Comment:
You don't need any predefined rules here; the application starts with an empty dictionary, on first request the dictionary is populated by parsing the validate.xml and any subsequent requests use the dictionary, until the application is destroyed.